docs/assets/scss/_admonition.scss

```diff
@@ -210,5 +210,25 @@ main {
         background-image: url(/icons/admonition-tip.svg);
       }
     }
+
+    .admonition.lead {
+      background: rgba(205, 239, 225, 0.4);
+      border: 1px solid #cdefe1;
+    
+      &::before {
+        width: 23px;
+        height: 24px;
+        left: 17px;
+        background-image: url(/icons/admonition-lead.svg);
```

Comment:
We don't want to specify the icon here. It should be configurable in the shortcode with default being `fa-light fa-magnifying-glass-arrow-right`

Reply:
@premkumr Icon is configurable now and default icon is set.